cordova-plugin-outline/android/java/org/outline/vpn/VpnTunnelService.java

```diff
@@ -173,6 +173,16 @@ public static TunnelConfig makeTunnelConfig(final String tunnelId, final JSONObj
     } catch (JSONException e) {
       LOG.fine("Tunnel config missing name");
     }
+    try {
```

Comment:
Can we get rid of `TunnelConfig` and `makeTunnelConfig`?

Ideally we would stick to the JSONObject (perhaps proto in the future) and treat it as opaque in the platform-specific code.

Reply:
I think that's a good idea, but I don't want to combine that with this change.